src/ts/content/extensions/util/onLogged.ts

```diff
@@ -17,9 +18,11 @@ interface OnLogOptions {
 	description?: string;
 }
 
+type LoggedStatus = {authed: boolean; sheetSet: boolean; ready: boolean};
 type OnLoggedCallback = () => void;
-const loginCallbacks: OnLoggedCallback[] = [];
-const logoutCallbacks: OnLoggedCallback[] = [];
+type InternalOnLoggedCallback = (status: LoggedStatus) => void;
```

Comment:
one takes a status and one doesn't... but why?

Reply:
From outside this file, no one should know about `status`!!